.github/workflows/changelog.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       #   or add `changelog/no-changelog` label if no release note is needed.
       - name: Ensure release note added
         # Only run this on pull requests
-        if: github.event_name == 'pull_request'
+        if: github.event_name == 'pull_request' && (startsWith(github.event.head_commit.message, 'fix') || startsWith(github.event.head_commit.message, 'feat'))
```

Comment:
I don't usually put the type indicator in commit messages. Maybe checking the pull request title would be more aligned with how we use pull request type indicators.

Reply:
Good point. I couldn't find a way of getting hold of the PR title, but I didn't look too deeply into it.

Reply:
Maybe `github.event.pull_request.title`? from: https://stackoverflow.com/a/71692835

Reply:
Ah great, thanks, didn't spot that!